avogadro/core/molecule.cpp

```diff
@@ -31,7 +31,7 @@ namespace Avogadro {
 namespace Core {
 
 Molecule::Molecule()
-  : m_graphDirty(false), m_basisSet(nullptr), m_unitCell(nullptr)
+  : m_graphDirty(true), m_basisSet(nullptr), m_unitCell(nullptr)
```

Comment:
If it's an empty molecule, by definition the graph can't be dirty.

Reply:
This was just to double check the RWMolecule problem, but yeas by definition should be false.